### PR TITLE
HEEDLS-326 - Tutorial Page - Change Button Text On Restart

### PR DESCRIPTION
**Pull request by SChandarana**
image

### CODE REVIEW

File: DigitalLearningSolutions.Web/ViewModels/LearningMenu/TutorialViewModel.cs [hunk 1]
```diff
@@ -89,6 +91,8 @@ int sectionId
                 tutorialInformation.DiagnosticAssessmentCompletionThreshold,
                 tutorialInformation.TutorialsCompletionThreshold
             );
+            TutorialStartButtonColour = tutorialInformation.Status == "Complete" ? "nhsuk-button--secondary" : "";
```
Comment (FridaTveit) [inline DigitalLearningSolutions.Web/ViewModels/LearningMenu/TutorialViewModel.cs:94]: I think `TutorialStartButtonStyling` or even `TutorialStartButtonAdditionalStyling` might be a more descriptive name? 🙂

**Review by FridaTveit — COMMENTED** [review]
One small suggestion, otherwise looks good 🙂

File: DigitalLearningSolutions.Web/ViewModels/LearningMenu/TutorialViewModel.cs (hunk 1 above)
Comment (SChandarana) [reply DigitalLearningSolutions.Web/ViewModels/LearningMenu/TutorialViewModel.cs:94]: Thanks! I was fighting with a variable name for that and couldn't for the life of me think of one aha

**Review by FridaTveit — APPROVED** [review]
Looks good 🙂

**Pull request merged**